pretix_sepadebit/payment.py

```diff
@@ -114,6 +118,33 @@ def settings_form_fields(self):
                                  'configuring at least 7 days.'),
                      min_value=1
                  )),
+                ('earliest_due_date',
+                 forms.DateField(
+                     label=_('Earliest debit due date'),
+                     help_text=_('Earliest date the direct debit can be due. '
+                                 'This date is used as the direct debit due date, if the order date plus pre-notification time would result in a due date earlier than this.'
+                                 'Customers with orders using the earliest due date, will receive an email informing them about the upcoming due date pre-notification days beforehand.'),
```

Comment:
Missing space, I think the commas are wrong and the last sentence is hard to read. 

cc @julia-luna for English copy editing :)

```suggestion
                                 'This date is used as the direct debit due date if the order date plus pre-notification time would result in a due date earlier than this. '
                                 'Customers with orders using the earliest due date will receive an email reminding them about the upcoming charge based on the configured pre-notification days.'),
```